### PR TITLE
flaky text fixes unit and e2e

### PR DESCRIPTION
**Pull request by avzz-19**

### CODE REVIEW

File: cypress/tests/dynamic-scan.spec.ts [hunk 1]
```diff
@@ -259,23 +262,27 @@ describe('Dynamic Scan', () => {
 
                 if (type !== 0) {
```
Comment (Yibaebi) [inline cypress/tests/dynamic-scan.spec.ts:263]: What does `0` mean here?

File: cypress/tests/dynamic-scan.spec.ts [hunk 2]
```diff
@@ -259,23 +262,27 @@ describe('Dynamic Scan', () => {
...
-                  cy.document()
-                    .findByText(cyTranslate('savedPreferences'))
-                    .should('exist');
+                  dynamicScanActions.chooseDevicePreferenceOption(
+                    '@deviceTypeSelect',
+                    cyTranslate('anyDevice')
+                  );
                 }
+
+                if (version !== '0') {
```
Comment (Yibaebi) [inline cypress/tests/dynamic-scan.spec.ts:271]: Here also. What does the `'0'` check stand for?

File: cypress/tests/dynamic-scan.spec.ts (hunk 1 above)
Comment (future-pirate-king) [reply cypress/tests/dynamic-scan.spec.ts]: changed it to a constant `ANY_DEVICE`

File: cypress/tests/dynamic-scan.spec.ts (hunk 2 above)
Comment (future-pirate-king) [reply cypress/tests/dynamic-scan.spec.ts]: changed it to a constant `ANY_VERSION`

**Review by Yibaebi — APPROVED** [review]

**Pull request merged**